### PR TITLE
adds disabled button state

### PR DESCRIPTION
**Pull request by leejordan**
screen shot 2016-01-21 at 11 05 14

### CODE REVIEW

File: less/modules/buttons.less [hunk 1]
```diff
@@ -24,12 +25,40 @@
...
+
+// disabled attribute can be used on buttons
+// .disabled class should be used on links as links do not have a disabled attribute
+
+.button {
+    &.button--disabled,
+    &[disabled] {
+        pointer-events: none;
+        .button-style(@colour-grey-lightest, @colour-grey-light, @colour-grey-lighter, transparent);
+        #gradient > .vertical(transparent, transparent);
```
Comment (notlee) [inline less/modules/buttons.less]: Are we using ids for increased specificity? O.o

Comment (leejordan) [inline less/modules/buttons.less]: That is not an id reference, it is a way of scoping mixins in LESS:

http://lesscss.org/features/#detached-rulesets-feature-scoping

Comment (notlee) [inline less/modules/buttons.less]: Oh of course, that wouldn't be valid! Ah interesting. I didn't know about that. :+1:

**Comment by notlee** [comment]
@leejordan approved!

**Pull request merged**